AI_POLICY.md

```
@@ -0,0 +1,88 @@
+# AI Policy
+
+### Purpose
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)